### ISSUE TITLE
Idea: Aliases on Tidal accounts

### ISSUE THREAD

**Issue by renebk** [michaelherger/lms-plugin-tidal#74]
When having multiple Tidal account added to the plugin, I'm my case they are either presented by email or msisdn.

It would be neat, if an alias or account name could be added. 

It's icing on the cake, low priority :)

![Screenshot_20240408_055806_LMS](https://github.com/michaelherger/lms-plugin-tidal/assets/10727684/d135ab2b-7066-4c13-8bbd-fdf8afd030ad)

**Comment by michaelherger** [comment]
TIDAL allows you to define first/last name. Did you set either of these?

I also see a "nickname" in the data, but have no clue where to define that.

**Comment by renebk** [comment]
Yes, I set both the First and Last Name on the tidal account. So, for my account I have: Username <MSISDN> and then First and Last Name

**Comment by renebk** [comment]
Thanks Michael. So, I tried quite some ways to add the First Name and Last Name to the Tidal account. Neither App or web-player allows you to provide this information. You can set a "Profile Name" in Webplayer and also in Android App. This is shared across Apps and Web-player. Android App also allows you to set a First Name.  On the "My Account" page, you can add a First and Last Name, which is used to generate a "Full Name". 
However, it does not seem that any of this information is shared on the API calls. Only account username and email is present. 

My dirty work around was to manually add the information (Full Name) to the tidal prefs file after setting up the accounts - works like a charm :)

**Comment by michaelherger** [comment]
What if you removed the account and added it back to the plugin? Would the data then show up?

Don't forget to back up your manually configured prefs file 😁.

**Comment by renebk** [comment]
Good point - less crying more backup ;-)

Unfortunately no. I tried all the sneaky paths, including remove/adding the accounts.
